src/services/waku.ts

```diff
@@ -0,0 +1,140 @@
+import { v4 as uuid } from "uuid";
+import {
+  PUBSUB_TOPIC,
+} from "@/constants";
+import { http } from "@/utils/http";
+
+export type Message = {
+  payload: string,
+  contentTopic: string,
+  version: number,
+  timestamp: number
+};
+
+type EventListener = (event: CustomEvent) => void;
+
+const SECOND = 1000;
+const LOCAL_NODE = "http://127.0.0.1:8645/";
+const FILTER_URL = "/filter/v2/";
+const LIGHT_PUSH = "/lightpush/v1/";
```

Comment:
can we use /relay/v1/messages/{pubsubTopic} instead? its assumed that the underlying node is a full node (have relay capabilities).

Reply:
should be solved here https://github.com/waku-org/waku-frontend/pull/10